meadow-client/src/MainFrame.purs

```diff
@@ -473,6 +432,25 @@ evalF (Undo next) = do
         Nothing -> ms
         Just netail -> netail
 
+evalF (HandleBlocklyMessage msg next) = do
+  case msg of
+    Initialized -> pure unit
+    (CurrentCode code) -> do
```

Comment:
I'd roll this pattern match into `evalF` m'self.

```
evalF (HandleBlocklyMessage Initialized next) = pure unit

evalF (HandleBlocklyMessage (Current code) next) = do
  ...
```

Reply:
done